lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -82,13 +82,19 @@ def register_offense(comment, matched_keyword)
 
         def offensive?(comment)
           line = source_line(comment)
+          return false if rbs_inline_annotation?(line, comment)
+
           KEYWORD_REGEXES.any? { |r| r.match?(line) } &&
             ALLOWED_COMMENT_REGEXES.none? { |r| r.match?(line) }
         end
 
         def source_line(comment)
           comment.source_range.source_line
         end
+
+        def rbs_inline_annotation?(line, comment)
+          comment.text.start_with?('#:') && line =~ /^\s*def\s/
```

Comment:
It can probably be done as follows:

```suggestion
          comment.text.start_with?('#:') && line.start_with?(/\A\s*def\s/)
```

And, is `\s*` necessary before `def` in this regexp? If so, please add test cases for it.